scripts/psiphon_install.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+PSIPHON_PATH=$HOME
+PSIPHON_PYCLIENT_PATH=$PSIPHON_PATH/psiphon-circumvention-system/pyclient
+
+sudo apt-get install mercurial
+cd $PSIPHON_PATH
+hg clone https://bitbucket.org/psiphon/psiphon-circumvention-system
+# optional, compile their ssh
+#cd psiphon-circumvention-system/Server/3rdParty/openssh-5.9p1/
+#./configure
+#make
+#cp ssh ../../../pyclient/
+#cd $PSIPHON_PATH
+
+# create psi_client.dat
+cat <<EOF > psi_generate_dat.py
+#!/usr/bin/env python
+
+import wget
+import os
+import json
+
+# Delete 'server_list' if exists
+if os.path.exists("server_list"):
+    # os.remove("server_list")
+    # os.rename("server_list", "server_list")
+    pass
+else:
+    # Download 'server_list'
+    url ="https://psiphon3.com/server_list'
+    wget.download(url)
+
+# convert server_list to psi_client.dat
+dat = {}
+dat["propagation_channel_id"] = "FFFFFFFFFFFFFFFF"
+dat["sponsor_id"] = "FFFFFFFFFFFFFFFF"
+dat["servers"] = json.load(open('server_list'))['data'].split()
+json.dump(dat, open('psi_client.dat', 'w'))
```

Comment:
ah, i see what you mean. I think there's no other code from Psiphon for linux to do that, as apparently it's done in a different way with the Windows client (see https://groups.google.com/forum/#!topic/psiphon3-developers/cb8CW7Y98nI).